nixos/modules/services/misc/transfer-sh.nix

```diff
@@ -0,0 +1,422 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Reply:
How is this not needed? If I stake it out, I'll have to add `lib.` to every mkOption, don't I?

Reply:
consider `inherits` where you find yourself using `lib.*` excessively.
[this](https://nix.dev/guides/best-practices.html#with-scopes)  nix.dev page on best practices with `with` may be helpful
an issue (#208242) has been made to track inappropriate uses of `with` such as this.

this suggestion has not been resolved.